index.html

```diff
@@ -0,0 +1,75 @@
+ <!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" href="normalize.css">
+	<link rel="stylesheet" href="styles.css">
+	<link href="https://fonts.googleapis.com/css2?family=Work+Sans&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300&display=swap" rel="stylesheet">
+	<meta name="viewport" content="width=device-width">
+	<title>Gabriel Rowan - Fullstack Developer Website</title>
+</head>
+<body>
+	<nav>
+			<a class="navLink" href="#about">About</a>
+			<a class="navLink" href="#projects">Projects</a>
+			<a class="navLink" href="#contact">Contact</a>
+	</nav>
+	<div class="landingContainer">
+		<section class="landingSection">
+			<div class="landingIntro">
+				<h6 class="landingContent">Hi, my name is</h6>
+				<h1 class="landingContent">Gabriel Rowan</h1>
+				<h2 class="landingContent">I'm a full-stack developer</h2>
+			</div>
+			<div class="landingImage">
+				<img src="landingImage.jpg"/>
+			</div>
+		</section>
+	</div>
+	<div class="aboutContainer" id="about">
+		<section class="aboutMe">
+			<h1>About Me</h1>
+			<p>I'm studying at the iO academy in Bath.<br><br> I like learning languages and Maths which is why I decided to train as a software developer.<br><br> 
+				As part of the course, I'm learning HTML, CSS, Javascript and PHP. </p>
+		</section>
+	</div>
+	<div class="projectContainer" id="projects">	
+		<section class="projectSection">
+				<span>
+					<h1>Projects</h1>
+				</span>
+			<div class="projectTable">
+				<div class="project specificProject">
+					<h2>Project 1</h2>
+					<p>I am a project. I am going to contain a lot of detail soon. Lots and lots of detail. </p>
+					<a href="https://github.com/gabrielrowan/gabrielrowan.github.io" target="_blank" rel="noreferrer noopener">Github Page</a>
+				</div>
+				<div class="project specificProject">
+					<h2>Project 2</h2>
+					<p>I am a project. I am going to contain a lot of detail soon. Lots and lots of detail.</p>
+					<a href="https://github.com/gabrielrowan/gabrielrowan.github.io" target="_blank" rel="noreferrer noopener">Github Page</a>
+					</div>
+				<div class="project">
+					<h2>Project 3</h2>
+					<p>I am a project. I am going to contain a lot of detail soon. Lots and lots of detail.</p>
+					<a href="https://github.com/gabrielrowan/gabrielrowan.github.io" target="_blank" rel="noreferrer noopener">Github Page</a>
+				</div>
+			</div>
+		</section>
+	</div> 
+	<div class="contactContainer" id="contact">
+		<section class="contactSection">
+			<h1>Contact</h1>
+			<p>If you'd like to get in touch, my email address is<br><br><br></p>
+			<p id="emailAddress">gabrielrowan82@gmail.com</p>
+			<p><br><br>To contact me on LinkedIn, please click below:<br><br>
+			</p>
+			<a class="linkedInLink" href="https://uk.linkedin.com/" target="_blank" rel="noreferrer noopener">LinkedIn</a>
+		</section>
+	</div>
+	<footer>
+		<p>© Copyright 2022 | Gabriel Rowan</p>
```

Comment:
Will this copyright symbol be supported in all browsers? Honestly not sure myself, but it could be a safer bet to use the below in its place (need to remove spaces for it to work) 
&#169; 
& # 1 6 9 ;